docs/BuildFromSource.md

```diff
@@ -43,9 +43,11 @@ The steps you follow next depend on your preferred development environment:
     > To modify an existing Visual Studio installation, [follow the instructions for installing from a configuration file](https://learn.microsoft.com/visualstudio/install/import-export-installation-configurations#import-a-configuration) and use the `.vsconfig` file located in the root of the repository:
 
     ```powershell
-    ./eng/scripts/InstallVisualStudio.ps1
+    ./eng/scripts/InstallVisualStudio.ps1 Enterprise
     ```
 
+    Replace `Enterprise` with `Community` if that is your preferred Visual Studio edition. If you use Visual Studio from a preview channel you can specify that also, for example `./eng/scripts/InstallVisualStudio.ps1 Enterprise Preview`.
+
```

Comment:
```suggestion
If you are seeing errors similar to `the imported proje
ct "aspnetcore.tools\msbuild\17.1.0\tools\MSBuild\Microsoft\VC\v170\Microsoft.Cpp.Default.prop
s" was not found`, this indicates you need to install/update Visual Studio by running `InstallVisualStudio.ps1`
```

Reply:
Whoa, the formatting on that got all messed up. But, something similar to this 😆